src/main.c

```diff
@@ -400,6 +414,145 @@ static void fill_grid(char *user_board, char *plain_board, int x_cursor, int y_c
 	}
 }
 
+char* get_paused_file_path(void) {
+	const char* home_path = getenv("HOME");
+	size_t len_home_path = strlen(home_path);
+
+	const char* dir_name = "/.nudoku/";
```

Comment:
I think we should use something like: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
Most likely: ` $XDG_STATE_HOME defines the base directory relative to which user-specific state files should be stored. If $XDG_STATE_HOME is either not set or empty, a default equal to $HOME/.local/state should be used. `